netweaver/init.sls

```diff
@@ -7,3 +9,6 @@ include:
   - netweaver.install_db
   - netweaver.install_pas
   - netweaver.install_aas
+  {% if netweaver.sap_host_exporter is sameas true %}
```

Comment:
What happens if the entry is not set? Does it work?
I would add the entry in the `defaults.yml`. It will make it easier.

Finally, could you add `-` to the `if` statements.This avoids the creation or new lines in the rendered file, what makes everything cleaner. Like:
`{%- if netweaver.sap_host_exporter is sameas true %}`
And
`{%- endif %}`

Reply:
done